ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward.hpp

```diff
@@ -80,17 +105,17 @@ struct ExecuteTernaryBackward {
         std::optional<Tensor> input_b_grad = std::nullopt) {
```

Comment:
can we make all optional tensors const reference too?

Reply:
Sure. These tensors (if one is passed) will be modified (in memory) during the op execution.

Reply:
Thinking about creating a reference in a common file outside of eltwise so that it can be used by all other ops in TTNN. also a common DefaultQueueId somewhere. Include this file across all ops